pkgs/development/python-modules/wacz/default.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  black,
+  boilerpy3,
+  cdxj-indexer,
+  click,
+  frictionless,
+  jsonlines,
+  pytest-cov,
+  pyyaml,
+  shortuuid,
+  typer,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "wacz";
+  version = "0.5.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "webrecorder";
+    repo = "py-wacz";
+    rev = "v${version}";
+    hash = "sha256-bGY6G7qBAN1Vu+pTNqRG0xh34sR62pMhQFHFGlJaTPQ=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "pytest-runner" ""
+  '';
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    black
+    boilerpy3
+    cdxj-indexer
+    click
+    frictionless
+    jsonlines
+    pytest-cov
+    pyyaml
+    shortuuid
+    typer
+  ];
+
+  optional-dependencies = {
+    # signing = [
+    #   authsign # not packaged
+    # ];
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # authsign is not packaged
+    "test_verify_signed"
+  ];
+
+  pythonImportsCheck = [
+    "wacz"
+  ];
+
+  dontCheckRuntimeDeps = true;
```

Comment:
I guess here is similar to cdxj-indexer only reason avoid check dependencies are typer requirements are too strict. Use `pythonRelaxDeps` instead.